src/components/career_card.jsx

```diff
@@ -25,12 +25,14 @@ const handleTouchEnd = (event) => {
 };
 
 const CareerCard = ({ career }) => (
-  <div
+  <div className="careerCard"
```

Comment:
I think it would be better to use kebab-case for the classes names - so for example instead of `careerCard` it would be `career-card`.

Reply:
sure. Will bear that in mind for the future.

Reply:
Perfect! Thanks @harrydry!

Reply:
@harrydry